style/javascript/sample.js

```diff
@@ -0,0 +1,7 @@
+object = { spacing: true }
+
+class Cat {
+  canBark() {
+    return false;
```

Comment:
I use semicolons to avoid having to know the [rules of ASI](http://bclary.com/2004/11/07/#a-7.9.1). In practice it's usually not a big deal but can sometimes lead to confusing bugs.